src/open_inwoner/accounts/models.py

```diff
@@ -216,6 +219,47 @@ class Meta:
         verbose_name = _("User")
         verbose_name_plural = _("Users")
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._old_bsn = self.bsn
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name} ({self.email})".strip()
```

Comment:
This isn't fully correct: because of legacy reasons we have `example.org` addresses as placeholders (see `User.get_contact_email`) that we don't want to display.